Mathlib/Algebra/Order/Floor/Defs.lean

```diff
@@ -304,7 +304,7 @@ private theorem int_floor_nonneg_of_pos [Ring α] [LinearOrder α] [FloorRing α
   int_floor_nonneg ha.le
 
 /-- Extension for the `positivity` tactic: `Int.floor` is nonnegative if its input is. -/
-@[positivity ⌊ _ ⌋]
+@[positivity ⌊_⌋]
```

Comment:
I'm not sure: if this change fine, or does the positivity attribute have any special quirks that make this a bad idea? (Same for the other changes below.)

Reply:
I found [other uses](https://github.com/leanprover-community/mathlib4/blob/53c38dde4d724f4b6b33f72d59bbb061b6ad45f9/Mathlib/Tactic/Positivity/Basic.lean#L331) of `positivity` that are similar and do not have the spaces.  I don't think that it matters, it is just a pretty-printed space.